### DISCUSSION TITLE
How can I bind `super+alt+h` on macOS?

### DISCUSSION THREAD

**Discussion by zx8** [ghostty-org/ghostty#3145] [Q&A] [answered]
I typically use this keybind for **h**orizontal splits, but it's causing the system-level `Hide Others` keybind to trigger.

I've tried to rebind it like:

```
keybind = super+alt+h=new_split:down
```

But the system-level keybind takes precedence.

My current workaround is to remap `super+alt+h` to `super+shift+alt+h` using [Karabiner](https://karabiner-elements.pqrs.org/), and then set `keybind = super+shift+alt+h=new_split:down`.

**Comment by ecdye** [comment]
You could try using the `global` option to see if that helps. See https://ghostty.org/docs/config/reference#keybind

**Reply by zx8** [reply]
The issue with `global` is that it applies across the entire system, even if Ghostty is not focused, right? I'd only want this bind to work when Ghostty is focused.

**Reply by ecdye** [reply]
True, but outside of changing the key bind in system settings to make it something different, I'm not sure if there is a way for Ghostty to override it right now.

**Comment by zx8** [comment, marked as answer]
Although this discussion came first, I'm closing it as a dupe of https://github.com/ghostty-org/ghostty/discussions/3187, since Mitchell has responded there.
